UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common.UnitTests/Helpers/FileShareServiceTests.cs

```diff
@@ -34,13 +33,14 @@ public class FileShareServiceTests
         private IOptions<CacheConfiguration> fakeCacheConfiguration;
 
         public string fakeFilePath = "C:\\HOME\\test.txt";
+        public string fakeLargeMediaZipFilePath = "D:\\HOME\\M01X01.zip";
         public string fakeFolderPath = "C:\\HOME";
         public string fakeZipFilepath = "D:\\UKHO\\V01X01";
         public string fakeExchangeSetPath = @"D:\UKHO";
         public string fakeBatchId = "c4af46f5-1b41-4294-93f9-dda87bf8ab96";
         public string fulfilmentExceptionMessage = "There has been a problem in creating your exchange set, so we are unable to fulfil your request at this time. Please contact UKHO Customer Services quoting error code : {0} and correlation ID : {1}";
```

Comment:
I know this is the fake endpoint but we need to consider the maintenance for this and possibly move this type of pattern out to environment type files